app/main.go

```diff
@@ -568,3 +596,11 @@ func setupLog(dbg bool) {
 	}
 	log.Setup(log.Msec, log.LevelBraces)
 }
+
+func removeTrainlingDot(name string) string {
```

Comment:
[strings.TrimSuffix](https://pkg.go.dev/strings#TrimSuffix) from stdlib should do it without a need in custom function and the test